packages/nimbus/src/components/tooltip/tooltip.tsx

```diff
@@ -1,31 +1,11 @@
-import { forwardRef } from "react";
-import { Tooltip as RATooltip } from "react-aria-components";
-import { TooltipRoot } from "./tooltip.slots";
-import type { TooltipProps } from "./tooltip.types";
+import { TooltipRoot } from "./components/tooltip.root";
+import { TooltipContent } from "./components/tooltip.content";
 
-/**
- * Tooltip
- * ============================================================
- * A contextual popup that displays a description for an element, uses `Tooltip` component from `react-aria-components`.
- *
- * Features:
- * - allows forwarding refs to the underlying DOM element
- * - accepts all native html 'HTMLDivElement' attributes (including aria- & data-attributes)
- * - allows overriding styles by using style-props
- *
- * Further Context:
- * - [React Aria Components Tooltip Documentation](https://react-spectrum.adobe.com/react-aria/Tooltip.html)
- * - [ARIA Tooltip Pattern](https://www.w3.org/TR/wai-aria-1.2/#tooltip)
- */
-export const Tooltip = forwardRef<HTMLDivElement, TooltipProps>(
-  function Tooltip({ children, placement = "bottom", ...props }, forwardedRef) {
-    return (
-      <TooltipRoot ref={forwardedRef} asChild {...props}>
-        <RATooltip placement={placement} {...props}>
-          {children}
-        </RATooltip>
-      </TooltipRoot>
-    );
-  }
-);
-Tooltip.displayName = "Tooltip";
+// Export individual components
+export { TooltipRoot, TooltipContent };
```

Comment:
I think we have a convention of prefixing these with an underscore to mark them as internal?

```suggestion
export { _TooltipRoot, _TooltipContent };
```

Reply:
Yes please, it's just a convention but then they switch categories in the [stats-overview](https://nimbus-documentation-git-fct-tooltip-api-commercetools.vercel.app/home/contribute/stats) and we'll have a better overview.

**From Components**
![image](https://github.com/user-attachments/assets/97b88b14-e349-4f81-8abb-276f74f173b9)

**To Types-Export**
![image](https://github.com/user-attachments/assets/80d9a1a1-744a-49b7-9e20-b9f34bc8b2e6)

Reply:
Resolved in 1ad8960f9de983e8904b2ee9950ffa8356557aa1